src/test/java/edu/cornell/kfs/concur/ConcurUtilsTest.java

```diff
@@ -1,99 +1,139 @@
 package edu.cornell.kfs.concur;
 
-import org.apache.commons.lang.StringUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.StringUtils;
 import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.kuali.kfs.sys.KFSConstants;
 
+import edu.cornell.kfs.concur.ConcurConstants.ConcurEventNotificationStatus;
+import edu.cornell.kfs.concur.ConcurConstants.ConcurEventNotificationType;
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotificationResponse;
+import edu.cornell.kfs.concur.services.ConcurAccountValidationTestConstants;
 import edu.cornell.kfs.sys.CUKFSConstants;
 
 public class ConcurUtilsTest {
     public static final String GOOD_EXPENSE_URI = "https://www.concursolutions.com/api/expense/expensereport/v1.1/reportfulldetails/123456578";
-    public static final String BAD_EXPENSE_URI = "https://www.concursolutions.com/api/someEndPoint";
-    
+    public static final String SOME_CONCUR_ENDPOINT_URI = "https://www.concursolutions.com/api/someEndPoint";
+
     public static final String GOOD_TRAVEL_REQUEST_URI = "https://www.concursolutions.com/api/travelrequest/v1.0/requests/1234567678";
-    public static final String BAD_TRAVEL_REQUEST_URI = "https://www.concursolutions.com/api/someEndPoint";
-    
+
     public static final String GOOD_CONCUR_FORMAT_CODE_AND_DESCRIPTION = "(1234567) some account description";
-    public static final String KFS_FORMAT_ACCOUNT_NUMBER = "1234567";   
+    public static final String KFS_FORMAT_ACCOUNT_NUMBER = "1234567";
     public static final String GOOD_CONCUR_FORMAT_CODE = "123";
-    
+
```

Comment:
the constant GOOD_CONCUR_FORMAT_CODE is unused?

Reply:
OK, I can remove